templates/project/src/main.rs

```diff
@@ -80,6 +81,7 @@ fn main() {
 
     if !storage.has_robot_configuration() {
         // check if any were statically compiled
+        // TODO: update with app address storage logic when version is incremented
```

Comment:
I'd suggest making a ticket with a fixVersion and adding the ticket into the TODO here as `TODO(RSDK-WXYZ): Update ...`.

Reply:
will do